packages/build/tests/install/snapshots/tests.js.md

```diff
@@ -81,8 +82,7 @@ Generated by [AVA](https://ava.li).
     ␊
     > Flags␊
       debug: true␊
-      featureFlags:␊
-        buildbot_esbuild: true␊
```

Comment:
The feature flag is no longer needed. Instead, people opt in to esbuild using `node_bundler = "esbuild"` in the functions configuration block.